client/src/components/Login.js

```diff
@@ -55,67 +57,36 @@ signupLink:{
     color:'#f50057'
 }
 }));
-function Login (){
+function Login (props){
     const classes = useStyles();
     const [email,setEmail]=useState('');
     const [password, setPassword]=useState('');
-    const [errors, setErrors]=useState({});
-
-    const validations = () => {
-        const errorsCopy = {...errors};
-        errorsCopy.email = email ? "" : "Please enter an email.";
-        errorsCopy.password = password ? "" : "Please enter a password.";
-        setErrors({ ...errorsCopy });
-
-        return Object.values(errorsCopy).every(field => field === "");
-    }
```

Comment:
We need to keep the error validation though

Reply:
I put required on textField instead to require user input.